keras/src/optimizers/muon.py

```diff
@@ -124,10 +124,9 @@ def __init__(
         self.ns_steps = ns_steps
         self.nesterov = nesterov
         self.exclude_embeddings = exclude_embeddings
-        # exclude_layers is a keyword at variable path
-        # so it must be a string
-        assert isinstance(exclude_layers, str) or exclude_layers is None
-        self.exclude_layers = exclude_layers.lower()
+        if exclude_layers is None:
+            exclude_layers = []
+        self.exclude_layers = exclude_layers
```

Comment:
Not very important, but these 3 lines are equivalent to one: `self.exclude_layers =  exclude_layers  or []`